### PR TITLE
LPS-182762 Add order type to CommerceOrder object

### PR DESCRIPTION
**Pull request by joanagrego**
Hi team, I'm sending this PR because we have a Marketplace demand where a need to add the `orderType` field to the  Commerce Order System Object.

Could you please review it?

Thanks.

### CODE REVIEW

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/object/system/CommerceOrderSystemObjectDefinitionManager.java [hunk 1]
```diff
@@ -129,6 +129,12 @@ public List<ObjectField> getObjectFields() {
 			createObjectField(
 				"Integer", "Integer", "order-status", "orderStatus", true,
 				true),
+			createObjectField(
+				"Text", "String", "orderTypeExternalReferenceCode",
+				"orderTypeExternalReferenceCode", true, true),
+			createObjectField(
+				"LongInteger", "Long", "order-type-id", "orderTypeId", true,
+				true),
```
Comment (carolmariaabb) [inline modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/object/system/CommerceOrderSystemObjectDefinitionManager.java:137]: @joanagrego to add these new fields correctly, you need to increase the version of this system object to 3, like we did [here](https://github.com/liferay/liferay-portal/commit/1960d0172f002a6a56a5125645a2146e00a757a2#diff-9e383eb4fdc0d8950c2709dde6043b95c64cd2e68a395d92fc1c4822fec4af07R130)

Comment (carolmariaabb) [inline modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/object/system/CommerceOrderSystemObjectDefinitionManager.java:137]: @joanagrego do we really need to set these new fields as `required`?

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/object/system/CommerceOrderSystemObjectDefinitionManager.java (hunk 1 above)
Comment (joanagrego) [reply modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/object/system/CommerceOrderSystemObjectDefinitionManager.java]: Done.

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:test:object

**Comment by carolmariaabb** [comment]
ci:test:sf

**Review by carolmariaabb — APPROVED** [review]
LGTM!

**Comment by carolmariaabb** [comment]
ci:forward:force

**Comment by carolmariaabb** [comment]
ci:forward:force

**Comment by carolmariaabb** [comment]
ci:forward:force

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by joanagrego** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**